R/doeResponseSurfaceMethodology.R

```diff
@@ -371,7 +371,11 @@ doeResponseSurfaceMethodology <- function(jaspResults, dataset, options, ...) {
   if (!dir.exists(dirname(outpath)))
     return()
 
-  utils::write.csv(x = design, file = options[["exportDesignFile"]], row.names = FALSE)
+  out_design <- design
+  colnames(out_design)[1:2] <- c("RunOrder", "StandardOrder")
+  out_design[["Response"]] <- ""
```

Comment:
if you want to remove the column, shouldn't it be
```suggestion
  out_design[["Response"]] <- NULL
```
?

Reply:
We don't want to remove the column right? We want to generate an empty column to display in the csv export so that the user can fill in the response.